highest_number_cubed/main.py

```diff
@@ -1,5 +1,9 @@
 def highest_number_cubed(limit):
-    pass
+    result = []
+    for i in range(1, limit):
+        if (i **3 < limit):
+            result.append(i)
+    return result.pop()
```

Comment:
Nice use of .pop :D

Reply:
Thank you! I have just learnt that Jessica is using something like "result[-1]". It's nice too!